src/main/java/com/aws/greengrass/clientdevices/auth/configuration/CDAConfiguration.java

```diff
@@ -22,6 +23,8 @@
  * <p>
  * Certificate Manager Service uses the following topic structure:
  * |---- configuration
+ * |    |---- security:
+ * |         |---- clientDeviceTrustDurationHours: "..."
```

Comment:
is hours the right unit we should use?

Reply:
I was thinking _seconds_ or _minutes_ would be too low level of granularity that the customers won't need. I updated the PR to `clientDeviceTrustDurationMinutes` now.